scripts/kernel_installer.py

```diff
@@ -0,0 +1,170 @@
+else:
+    print('New kernel version found')
+    # Step 0: Check if kernel already present in repo
+    with subprocess.Popen(["apt-get", "update"],
+                          stdout=subprocess.PIPE) as p:
+        for line in p.stdout:
+            print(line, end='\n') # process line here
+
+    with subprocess.Popen(["apt-cache", "search", f"{current_patch_hash}"],
+                          stdout=subprocess.PIPE) as packages:
+        # If present in repo - install from apt
+        if len(packages.stdout.readlines()) >= 3:
+            print('Kernel found, installing from repo')
+            with subprocess.Popen(["apt", "install", f"*5.10.35.tfw-{current_patch_hash}"],
+                          stdout=subprocess.PIPE) as p:
+                for line in p.stdout:
+                    print(line, end='\n')
+
+        # Else - Run local build
+        else:
+            print('Need to get another version of kernel')
+            print(f'{current_patch_hash} != {current_kernel_hash}')
+            
+            
+
+            # Step 1: Download sources
+            print('Step 1: Downloading 5.10.35 sources...')
+            url = 'https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/snapshot/linux-5.10.35.tar.gz'
+            r = requests.get(url, allow_redirects=True)
+            open('download.tar.gz', 'wb').write(r.content)
+            print('Done')
+
+            # Step 2: Untar
+            print('Step 2: Untar sources...')
+            my_tar = tarfile.open('download.tar.gz')
+            my_tar.extractall('.') # specify which folder to extract to
+            my_tar.close()
+            print('Done')
+
+            # Step 3: Apply EXTRAVERSION and patch
+            print('Step 3: Patch...')
+            with open('linux-5.10.35/Makefile') as f:
+                    data = f.read()
+                    data = data.replace('EXTRAVERSION =', f'EXTRAVERSION = .tfw-{current_patch_hash}')
+                    data = data.replace('NAME = Dare mighty things', f'NAME = Some test string')
+            with open(r'linux-5.10.35/Makefile', 'w') as file:
+                file.write(data)
+            shutil.copyfile(f'{tempesta_path}/linux-5.10.35.patch', 'linux-5.10.35/linux-5.10.35.patch')
+            with subprocess.Popen(["patch -p1 < linux-5.10.35.patch"],
+                                  stdout=subprocess.PIPE,
+                                  shell=True,
+                                  cwd=f"linux-5.10.35") as p:
+                output, err = p.communicate()
+                patch_out = output.decode("utf-8").strip()
+
+            # Step 4: config
+            if os.path.exists('.config'):
+                print('Step 4: Predefined .config found...')
+                subprocess.Popen(['cp', '.config', 'linux-5.10.35/config'], stdout=subprocess.PIPE)
+            else:
+                print('Step 4: Use OS .config...')
+                subprocess.Popen(['cp', f'/boot/config-{os.uname().release}', '.config'],
```

Comment:
If, for some reason, the current running kernel is much newer than the one we're trying to build, we might have problems reusing the config from `/boot`. We need to check that it's not `too new` :). 
Also, we should check `CONFIG_SECURITY_TEMPESTA` defines are present in `.config` as described [here](https://github.com/tempesta-tech/tempesta/wiki/Install-from-Sources)